### ISSUE TITLE
Save values as strings

### ISSUE THREAD

**Issue by pastorryanhayden** [aryehdesigns/statamic-location#3]
When I try to use this in conjunction with your (excellent) statamic-fetch plugin, I’m getting a whoops error from Laravel telling me that Laravel it’s expecting either a string or an integer.  The addon saves the negative Lat/Long values as a string, but saves the positive ones as a float and it’s breaking things.

**Comment by jdsimcoe** [comment]
@pastorryanhayden @aryehraber I'm having the same issue here. Any progress on this?

**Comment by aryehraber** [comment]
Haven't had a chance to look into it tbh, can you supply some example data that it saves?

**Comment by jdsimcoe** [comment]
Here are various points that are saved in the Markdown files with an attached `Location` field. I've anonymized it, but it conveys the idea.

`1.md`
```md
location:
  name: Anystreet USA
  latitude: 45.3123456
  longitude: -122.6543219
```
`2.md`
```md
location:
  name: Anotherstreet USA
  latitude: '45.498767'
  longitude: -122.8678906
```

**Comment by jdsimcoe** [comment]
FWIW, when I output the fields into a JSON array for the map, they come across as pure numbers. Just curious that some of the files are encoded with single quotes `'`.

**Comment by aryehraber** [comment]
Yeah, that was going to be my next suggestion, to use the `json` modifier which should ensure things are passed along correctly.

Thanks for sharing the data; looks like when Statamic does its thing on save it _sometimes_ converts to a string... I'm honestly not sure whether this is an issue on Statamic's end or within the Location addon (since the addon doesn't do the "saving" part).

Leave this issue open for now, I'll do my best to give it a test run at some point — though since I haven't needed this fieldtype in ages I'm not sure when I'll get to it.
